extensions/2.0/Vendor/EXT_lights_image_based/README.md

```diff
@@ -0,0 +1,127 @@
+
+This extension provides the ability to define image-based lights in a glTF scene. Image-based lights consist of an environment map that represents specular radiance for the scene as well as irradiance information.
+
+Many 3D tools and engines support image-based global illumination but the exact technique and data formats employed vary. Using this extension, tools can export and engines can import image-based lights and the result should be highly consistent. 
+
+This extension specifies exactly one way to format and reference the environment map to be used as well as one format for HDR image encoding. The goals of this are two-fold. First, it makes implementing support for this extension easier. Secondly, it ensures that rendering of the image-based lighting is consistent across runtimes.
+
+A conforming implementation of this extension must be able to load the image-based environment data and render the PBR materials using this lighting. 
+
+## Defining an Image-Based Light
+
+The EXT_lights_image_based extension defines an array of image-based lights at the root of the glTF and then each scene can reference one. Each image-based light definition consists of a single cubemap that describes the specular radiance of the scene, the l=2 spherical harmonics coefficients for diffuse irradiance and rotation and intensity values.
+
+```javascript
+"extensions": {
+    "EXT_lights_image_based" : {
+        "lights": [
+            {
+                "intensity": 1.0,
+                "rotation": [0, 0, 0, 1],
+                "irradianceCoefficients": [...3 x 9 array of floats...],
+                "specularImageSize": 256,
+                "specularImages": [
+                    [... 6 cube faces for mip 0 ...],
+                    [... 6 cube faces for mip 1 ...],
+                    ...
+                    [... 6 cube faces for mip n ...]
+                ],
+            }
+        ]
+    }
+}
+```
+
+## Prefiltered Specular Radiance Cubemaps
+
+The cubemap used for specular radiance is defined along with its prefiltered mipmaps with each face of the cube defined as separate images. The various mip levels evenly map to roughness values from 0 to 1 in the PBR material and should be generated with a principled multi-scatter GGX normal distribution. The data in the maps represents illuminance in lux (lm/m^2).
+
+The entire mip chain of images should not be provided. Instead, the lowest-resolution mip should have sufficient size to represent the maximally-blurred radiance map (say, 16x16) corresponding to roughness=1. The `specularImageSize` value defines the largest dimension of mip 0 and, taken together with the number of defined mips, should give the loading runtime the information it needs to generate the remainder of the mip chain and sample the appropriate mip level in the shader.
+
+*(say something about tooling for generating these mips)*
+
+Cube faces are defined in the following order and must be authored with the Y-axis inverted: 
```

Comment:
What does this mean?  Inverted with respect to what, glTF?

Should we specify face orientations, as well as axis assignments?  For example, for whichever face is above the model, how is the image on that face oriented?